index.html

```diff
@@ -2747,6 +2747,26 @@ <h4>Context Injection</h4>
 (`https://www.w3.org/ns/did/v1`).
           </p>
 
+          <pre class="example nohighlight" title="A controller document without an @context property">
+{
+  "id": "https://controller.example/101",
+  "verificationMethod": [{
+    "id": "https://controller.example/101#key-203947",
+    "type": "JsonWebKey",
+    "controller": "https://controller.example/101",
+    "publicKeyJwk": {
+      "kid": "key-203947",
+      "kty": "EC",
+      "crv": "P-256",
+      "alg": "ES256",
+      "x": "f83OJ3D2xF1Bg8vub9tLe1gHMzV76e8Tus9uPHvRVEU",
+      "y": "x_FEzRu9m36HLN_tue659LNpXW6pCyStikYjKIWI5a0"
+    }
+  }],
+  "authentication": ["#key-20240828"]
```

Comment:
Without this, the example would be invalid because the reference is to a document local VM that otherwise doesn't exist. A reference could be made to some other VM that lives in another document (if it uses an absolute URL) if we wanted, but this is a simple fix:

```suggestion
  "authentication": ["#key-203947"]
```